src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString
+    .split(';')
+    .map((styleString) => styleString.split(':').map((rule) => rule.trim()));
+
+  const formattedStyles = {};
+
+  for (const style of stylesList) {
+    formattedStyles[style[0]] = style[1];
```

Comment:
It's a good practice to check if `style[0]` and `style[1]` are defined before assigning them to `formattedStyles`. This will prevent potential errors if the input string has malformed style declarations.